src/components/Button.vue

```diff
@@ -5,7 +5,7 @@ import AIcon from './Icon.vue'
 
 type ButtonVariant = 'primary' | 'subtle' | 'standard'
 
-type ButtonSize = 'md' | 'lg' | 'auto'
+type ButtonSize = 'sm' | 'md' | 'lg' | 'auto'
```

Comment:
Shouldn't it be updated here:
<img width="799" alt="image" src="https://github.com/archilogic-com/honeycomb/assets/7584821/290e48de-a0d1-4a81-888e-eb76435f1614">